docs/apache-airflow-providers-sqlite/operators.rst

```diff
@@ -1,69 +0,0 @@
- .. Licensed to the Apache Software Foundation (ASF) under one
-    or more contributor license agreements.  See the NOTICE file
-    distributed with this work for additional information
-    regarding copyright ownership.  The ASF licenses this file
-    to you under the Apache License, Version 2.0 (the
-    "License"); you may not use this file except in compliance
-    with the License.  You may obtain a copy of the License at
-
- ..   http://www.apache.org/licenses/LICENSE-2.0
-
- .. Unless required by applicable law or agreed to in writing,
-    software distributed under the License is distributed on an
-    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-    KIND, either express or implied.  See the License for the
-    specific language governing permissions and limitations
-    under the License.
-
-
-
-.. _howto/operator:SqliteOperator:
-
-SQLExecuteQueryOperator to connect to Sqlite
```

Comment:
Why are we removing this?
It explains that to query SqlLite you should use SQLExecuteQueryOperator. That is information we should keep in the docs. Actually not only here but for all sql related providers that we replaced with SQLExecuteQueryOperator

Reply:
Do we still need to keep deprecated doc information `_howto/operator:SqliteOperator` ? is change log is enough?

Reply:
File name and titles may be adjusted but I think its nice to have example and information how to use SQLExecuteQueryOperator with SqlLite (and same for all others.. trino, presto, etc...)

It's also OK to cross refrence to SQLExecuteQueryOperator docs.

Reply:
Yeah, you are right - I deleted too much docs. Reverted and just changed the note.